docs/feature-store/using-spark-engine.md

```diff
@@ -238,4 +268,73 @@ target = ParquetTarget(
 )
 
 fstore.ingest(feature_set, source, targets=[target], run_config=run_config, spark_context=spark_service_name)
-```
\ No newline at end of file
+```
+
+## Spark ingestion from Snowflake example
+
+Spark ingestion from Snowflake can be executed both remotely and locally. The following code executes local data ingestion from Snowflake.
+
+```
+from pyspark.sql import SparkSession
+
+import mlrun
+import mlrun.feature_store as fstore
+from mlrun.datastore.sources import SnowflakeSource
+
+spark = SparkSession.builder.appName("snowy").getOrCreate()
+
+mlrun.set_environment(project="feature-store")
+feature_set = fstore.FeatureSet(
+    name="customer", entities=[fstore.Entity("C_CUSTKEY")], engine="spark"
+)
+
+source = SnowflakeSource(
+    "customer_sf",
+    query="select * from customer limit 100000",
+    url="<url>",
+    user="nnnnnn",
+    password="nnnnnn",
+    database="SNOWFLAKE_SAMPLE_DATA",
+    schema="TPCH_SF1",
+    warehouse="compute_wh",
+)
+
+fstore.ingest(feature_set, source, spark_context=spark)
+```
+
+## Spark ingestion from Azure example
+
+Spark ingestion from Azure can be executed both remotely and locally. The following code executes remote data ingestion from Azure.
+
+```
+# Initialize the MLRun project object
+project_name = "spark-azure-test"
+project = mlrun.get_or_create_project(project_name, context="./")
+
+from mlrun.runtimes import RemoteSparkRuntime
+RemoteSparkRuntime.deploy_default_image()
+
+from mlrun.datastore.sources import CSVSource
+from mlrun.datastore.targets import ParquetTarget
+from mlrun import code_to_function
+import mlrun.feature_store as fstore
```

Comment:
This line is redundant assuming we return the `import mlrun.feature_store as fs` line to where it belongs. 
However, My suggestion is to do the following:
* Only keep this line (remove the `import mlrun.feature_store as fs` line altogether)
* Make sure only `fstore.` is used - there's a single place to change - see next comment.